src/common/repository/UserRepository.ts

```diff
@@ -0,0 +1,37 @@
+import { Admin } from 'common/utils';
+
+const KEY_TOKEN = 'AUTH_TOKEN';
+const KEY_USER = 'ADMIN_INF';
+
+export const UserRepository = {
+	setToken: (value: any) => {
+		localStorage.setItem(KEY_TOKEN, value);
+	},

```

Comment:
creo que hay una nueva sintaxis para este tipo de cosas

```suggestion
	setToken(value: any) {
		localStorage.setItem(KEY_TOKEN, value);
	},

```

Reply:
Para los métodos de localstorage? voy a investigar

Reply:
nono, en un objeto js, si tienes un key cuyo valor es una funcion
```js
{
  foo: () => { 
    console.log('hola');
  }
}
```

Puedes ponerla sin los dos puntitos. Es la misma sintaxis que habia cuando hacias funciones en un Class component antes que hubieran hooks en react

```js
{
  foo() { 
    console.log('hola');
  }
}
```

Reply:
He visto que los métodos que se usan para el localstorage son los mismos o a qué te refieres?

Reply:
a la sintaxis, no al localstorage, no queda claro mi ejemplo?

Reply:
ohh ya entendí dale ahora lo cambio, gracias